projects/dotcms/src/lib/api/DotApiContent.ts

```diff
@@ -22,12 +45,12 @@ export class DotApiContent {
 
     private async doRequest<Content extends DotCMSContent>(
         url: string,
-        params: Content
+        params?: Content
     ): Promise<Response> {
         const response = await this.dotCMSHttpClient.request({
             url,
-            method: 'POST',
-            body: JSON.stringify(params)
+            method: params ? 'POST' : 'GET',
```

Comment:
This is not a good approach, conceptually it doesn't make sense that because we sent or not `params` the method change.